### ISSUE TITLE
Commit history

### ISSUE THREAD

**Issue by yalinli2** [BioSTEAMDevelopmentGroup/biosteam#40]
Looks like the commit history has been reset (and it has been done so in the past), is there a reason for this? Doing so will cause some problem when I try to `git pull upstream`.  Thanks!

**Comment by yoelcortes** [comment]
Yes, the commit history included very big files that are not relevant anymore, major changes from biosteam 1 to biosteam 2, and poor notes like "update" in most of the first hundred commits. Because the commits are so big, it would take a good hour for someone to clone the repository.

I'm not sure how to fix the `git pull upstream` problem, but you could either delete and reclone the repository or I can try to help if you post the error here. Thanks, and sorry for the inconvenience!

**Comment by yalinli2** [comment]
Oh no problem, actually resetting the history helped a lot in reducing the size

It *looks like* the `git pull upstream` problem is a one-time thing, I just need to allow unrelated histories and I think I've got it figured out.

PS, I think a lot of the commits were added the last time you merged my PR about the carbon capture units. Since I didn't reset the commit history at that time, my fork had all the commit history from the very beginning. I also reset the commit history on my fork so in the future even if my fork is merged into the root repository, it shouldn't add a lot, thanks!
